curated_transformers/generation/logits.py

```diff
@@ -121,3 +122,43 @@ def _process_logits(self, logits: Tensor):
             return
 
         logits /= self.temperature
+
+
+class MaskTransform(LogitsTransform):
+    """
+    Set the probability of specific classes to zero.
+    """
+
+    def __init__(self, classes_to_mask: List[int]):
```

Comment:
```suggestion
    def __init__(self, classes_to_mask: List[int]):
```

Probably `Set[int]` or if we want to be more generic `Iterable[int]`.

Reply:
Switched to `Iterable`